src/main/java/se/kth/meta/wscomm/message/FieldsMessage.java

```diff
@@ -90,7 +90,7 @@ public String getAction() {
     } catch (NullPointerException e) {
     }
     try {
-      //sanitize fucking maxsize in case the user has entered shit
+      //sanitize maxsize in case the user has entered shit
```

Comment:
Nice comment refactoring. The nice word for "shit" would be "gibberish", or something like that :D